sunpy/map/mapbase.py

```diff
@@ -1829,6 +1839,13 @@ def draw_rectangle(self, bottom_left, *, width: u.deg = None, height: u.deg = No
 
         width = Longitude(top_right.spherical.lon - bottom_left.spherical.lon)
         height = Latitude(top_right.spherical.lat - bottom_left.spherical.lat)
+        if width < 0:
+            warnings.warn("The rectangle is inverted in the left/right (longitude) direction,  "
```

Comment:
```suggestion
            warnings.warn("The rectangle is inverted in the left/right direction,  "
```

Reply:
In `draw_rectangle` width is hard-coded to apply to the longitude direction (see the above two lines), so having longitude here is correct. I can still remove if it would make for a better warning?